index.html

```diff
@@ -30,12 +30,100 @@ <h2>I’am a Software Developer</h2>
+            <hr/>
+            <section class="works">
+                <article class="storie">
+                    <img src="images/stories.png" alt="stories image" />
+                    <h2>Multi-Post Stories</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    <ul class="tools">
+                        <li>css</li>
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>ruby</li>
+                    </ul>
+                    <button type="button" class="button">See Project</button>
+                </article>
+                <article class="art">
+                    <h2>Profesional Art Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul class="tools1">
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>ruby</li>
+                    </ul>
+                    <button type="button" class="button">See Project</button>
+                </article>
+                <article class="art">
+                    <h2>Profesional Art Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul class="tools1">
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>ruby</li>
+                    </ul>
+                    <button type="button" class="button">See Project</button>
+                </article>
+                <article class="art">
+                    <h2>Profesional Art Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul class="tools1">
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>ruby</li>
+                    </ul>
+                    <button type="button" class="button">See Project</button>
+                </article>
+    
+            </section>
+        </div>
+        <section class="aboutme">
+            <h2>About me</h2>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+           
+            <div class="block-img-group">
+                <button type="button" class="button">Get My Resume</button>
+                <img class="img2" src="images/Group.png" alt="Resume"></div>
```

Comment:
- [ ] The background of this section should set using CSS using `background-image` attribute and not using an `img` tag, kindly fix that. 😁😁

If you want to know more about how to use `background-image` [check this quick guide](https://www.w3schools.com/cssref/pr_background-image.asp).